src/webRequest.js

```diff
@@ -0,0 +1,43 @@
+export const doGet = (url) => fetch(url)
+  .then((response) => {
+    try {
+      return response.json();
+    } catch {
+      let ansData = {};
+      if (response.status === 201) {
+        ansData = {
+          status: true,
+        };
+      } else {
+        ansData = {
+          status: false,
+        };
+      }
+      return ansData;
+    }
+  });
+
+export const doPost = (url, payload = null) => fetch(url, {
+  method: 'POST',
+  body: JSON.stringify(payload),
+  headers: {
+    'Content-type': 'application/json; charset=UTF-8',
+  },
+})
+  .then(async (response) => {
+    let ansData = {};
+    try {
+      ansData = await response.json();
+    } catch {
+      if (response.status === 201) {
+        ansData = {
+          status: true,
+        };
+      } else {
+        ansData = {
+          status: false,
+        };
+      }
+    }
+    return ansData;
+  });
```

Comment:
- [ ] Please consider refactoring this code to use ES6 features such as `async` and `await` instead of `.then().catch()` chains! 🤓 🔧